### PR TITLE
fix: invalid INTERNAL_IP environment value

### PR DESCRIPTION
**Pull request by angwdev**
If this is not a suitable solution, please correct me. 

```
ip addr show $(ip route | awk '/default/ { print $5 }') | grep "inet" | head -n 1 | awk '/inet/ {print $2}' | cut -d'/' -f1
```

### CODE REVIEW

**Comment by DaneEveritt** [comment]
cc @matthewpi @Software-Noob can either of you look at this?

**Comment by trenutoo** [comment]
It is indeed broken for newer `ip` outputs that append the uid as that'll be returned instead of the IP. It could be easily fixed by ignoring the appended UID `ip route get 1 | awk '{print $(NF-2);exit}'`

**Comment by angwdev** [comment]
Done

File: games/source/entrypoint.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ TZ=${TZ:-UTC}
 export TZ
 
 # Set environment variable that holds the Internal Docker IP
-INTERNAL_IP=$(ip route get 1 | awk '{print $NF;exit}')
+INTERNAL_IP=$(ip addr show $(ip route | awk '/default/ { print $5 }') | grep "inet" | head -n 1 | awk '/inet/ {print $2}' | cut -d'/' -f1)
```
Comment (trenutoo) [inline games/source/entrypoint.sh]: You forgot to change this

File: oses/debian/entrypoint.sh [hunk 2]
```diff
@@ -26,7 +26,7 @@ TZ=${TZ:-UTC}
 export TZ
 
 # Set environment variable that holds the Internal Docker IP
-INTERNAL_IP=$(ip route get 1 | awk '{print $NF;exit}')
+INTERNAL_IP=$(ip addr show $(ip route get 1 | awk '{print $(NF-2);exit}')
```
Comment (trenutoo) [inline oses/debian/entrypoint.sh]: This is not right. You have leftover IP a here

**Review by trenutoo — APPROVED** [review]
LGTM, thank you for contributing and noticing the issue :+1:

**Pull request merged**